src/components/screens/editor/editor-screen.styles.ts

```diff
@@ -17,6 +17,74 @@ export const useEditorScreenStyles = makeStyles({
   editorContainer: {
     // 100% height substracted with two toolbars and extra spacing for tight fit
     height: `calc(100% - ${toolbarHeight}px - ${toolbarHeight}px - ${theme.spacing(2)}px)`,
+  },
+  divider: {
+    marginTop: theme.spacing(2),
+    marginBottom: theme.spacing(2)
+  },
+  button: {
+    marginTop: theme.spacing(2),
+    marginBottom: theme.spacing(1),
+    marginLeft: 0,
+    marginRight: 0,
+    paddingTop: 9.5,
+    paddingBottom: 9.5,
+    width: "100%",
+    color: "primary"
```

Comment:
this does not work. use theme.palette.primary.main to access the theme primary color

Reply:
Or if you want to use primary color variant it is best to do it in the button component itself: <Button color="primary">